apps/emqx/test/emqx_cm_SUITE.erl

```diff
@@ -109,9 +109,13 @@ t_open_session(_) ->
     emqx_cm:unregister_channel(<<"clientid">>),
     ok = meck:unload(emqx_connection).
 
+rand_client_id() ->
+    list_to_binary("client-id-" ++ integer_to_list(erlang:system_time())).
```

Comment:
just feel `erlang:unique_integer` may be better here
```suggestion
    list_to_binary("client-id-" ++ integer_to_list(erlang:unique_integer([positive]))).
```

Reply:
system_time is unique enough for tests.